packages/wonder-blocks-modal/components/one-column-modal.js

```diff
@@ -27,6 +32,58 @@ type Props = {
     onClickCloseButton: () => void,
 };
 
+class ContentWrapper extends React.Component<Props> {
+    static defaultProps = {
+        onClickCloseButton: () => {},
+    };
+
+    static contextTypes = {
+        gridSize: propTypes.string,
+    };
+
+    render() {
+        const {onClickCloseButton, content, footer} = this.props;
+        const {gridSize} = this.context;
+
+        if (gridSize !== "small") {
+            return (
+                <View style={styles.contentWrapper}>
+                    <ModalContentPane
+                        showCloseButton
+                        onClickCloseButton={onClickCloseButton}
+                        content={content}
+                        footer={footer}
+                    />
+                </View>
+            );
+        }
+
+        return (
+            <View style={styles.contentFooterWrapper}>
```

Comment:
I like this separation.  I wonder if we should maybe split this component into `MobileModalContent` and `DesktopModalContents` and have `OneColumnModal` decide which to render.  The benefit is that we'd be able to render both the desktop and mobile versions of the modals in styleguidist at the same time and people wouldn't have to resize the browser to see each version.

Reply:
Hmmm - it would be quite as easy as that as we'd also have to render the correct ModalDialog, too (and a lot of the styling is determined by the viewport/grid so we'd have to find a way to override that). I'd like to defer that for now, even though it would be nice, as it's certainly going to be pretty challenging.

Reply:
I see... nevermind then.  I guess people will have to resize their browsers.